src/EditorFeatures/Core.Wpf/WpfClassificationExtensions.cs

```diff
@@ -18,12 +18,17 @@ internal static partial class WpfClassificationExtensions
     {
         public static Run ToRun(this ClassifiedText part, IClassificationFormatMap formatMap, ClassificationTypeMap typeMap)
         {
-            var text = part.Text;
+            var run = new Run(part.Text);
 
-            var run = new Run(text);
+            var classificationType = typeMap.GetClassificationType(part.ClassificationType);
+            if (classificationType == null)
```

Comment:
can you clarify when this happens?  All our classification types should be in the typemap...